packages/@react-aria/menu/src/useMenu.ts

```diff
@@ -63,6 +63,7 @@ export function useMenu<T>(props: AriaMenuOptions<T>, state: TreeState<T>, ref:
     selectionManager: state.selectionManager,
     collection: state.collection,
     disabledKeys: state.disabledKeys,
+    preventEscapeClearsSelection: true,
```

Comment:
Now that I think about this, if menu is the only place where this is the case, we could consider overriding the keyboard handler and not forwarding escape.

Otherwise, this prop name doesn't really fit our pattern... `disallowClearSelection` or `disallowEscape` might be closer (like `disallowEmptySelection` and `disallowSelectAll`) but still sorta weird.

Reply:
I think I like not introducing a new prop for the moment more. If we find two+ instances where this is needed, then I think we could revisit adding it